xla/service/gpu/fusions/mlir/erase_dead_functions.cc

```diff
@@ -77,8 +77,7 @@ class EraseDeadFunctionsPass
 
 }  // namespace
 
-std::unique_ptr<mlir::OperationPass<mlir::ModuleOp>>
-CreateEraseDeadFunctionsPass() {
+std::unique_ptr<mlir::Pass> CreateEraseDeadFunctionsPass() {
```

Comment:
I pushed this after the approval to resolve a linking error. The header uses `mlir::Pass` and MSVC appears to be sensitive to that resulting in mismatching symbol names that fail to link.